reference/lib/ReferenceAmountDeriver.sol

```diff
@@ -35,16 +34,24 @@ contract ReferenceAmountDeriver is AmountDerivationErrors {
     function _locateCurrentAmount(
         uint256 startAmount,
         uint256 endAmount,
-        uint256 elapsed,
-        uint256 remaining,
-        uint256 duration,
+        uint256 startTime,
+        uint256 endTime,
         bool roundUp
-    ) internal pure returns (uint256) {
+    ) internal view returns (uint256) {
```

Comment:
Be sure to update the natspec from “Internal pure function…” to “Internal view function…” for the functions that have the new visibility